{"sections":[],"identifier":{"url":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/AsyncThrows","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Represents an asynchronous method that can also throw an error."}],"relationshipsSections":[{"kind":"relationships","type":"conformsTo","title":"Conforms To","identifiers":["doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Asynchronous","doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Effect","doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Throwing"]}],"metadata":{"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AsyncThrows","kind":"identifier"}],"modules":[{"name":"SwiftMocking"}],"title":"AsyncThrows","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AsyncThrows"}],"externalID":"s:12SwiftMocking11AsyncThrowsO","symbolKind":"enum","roleHeading":"Enumeration"},"hierarchy":{"paths":[["doc:\/\/SwiftMocking\/documentation\/SwiftMocking"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftmocking\/asyncthrows"]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AsyncThrows"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","references":{"doc://SwiftMocking/documentation/SwiftMocking/Effect":{"role":"symbol","fragments":[{"text":"protocol","kind":"keyword"},{"kind":"text","text":" "},{"text":"Effect","kind":"identifier"}],"identifier":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Effect","navigatorTitle":[{"text":"Effect","kind":"identifier"}],"abstract":[{"type":"text","text":"A protocol that defines the effect of a method, such as throwing an error or being asynchronous."}],"url":"\/documentation\/swiftmocking\/effect","type":"topic","title":"Effect","kind":"symbol"},"doc://SwiftMocking/documentation/SwiftMocking":{"url":"\/documentation\/swiftmocking","type":"topic","kind":"symbol","title":"SwiftMocking","identifier":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking","abstract":[],"role":"collection"},"doc://SwiftMocking/documentation/SwiftMocking/AsyncThrows":{"role":"symbol","url":"\/documentation\/swiftmocking\/asyncthrows","kind":"symbol","title":"AsyncThrows","abstract":[{"type":"text","text":"Represents an asynchronous method that can also throw an error."}],"identifier":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/AsyncThrows","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AsyncThrows","kind":"identifier"}],"navigatorTitle":[{"text":"AsyncThrows","kind":"identifier"}]},"doc://SwiftMocking/documentation/SwiftMocking/Asynchronous":{"kind":"symbol","abstract":[],"type":"topic","role":"symbol","identifier":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Asynchronous","fragments":[{"kind":"keyword","text":"protocol"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Asynchronous"}],"title":"Asynchronous","navigatorTitle":[{"text":"Asynchronous","kind":"identifier"}],"url":"\/documentation\/swiftmocking\/asynchronous"},"doc://SwiftMocking/documentation/SwiftMocking/Throwing":{"identifier":"doc:\/\/SwiftMocking\/documentation\/SwiftMocking\/Throwing","role":"symbol","title":"Throwing","abstract":[],"url":"\/documentation\/swiftmocking\/throwing","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"Throwing"}],"fragments":[{"text":"protocol","kind":"keyword"},{"kind":"text","text":" "},{"text":"Throwing","kind":"identifier"}],"type":"topic"}}}